src/components/accessories/customModal/CustomModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Modal from "@material-ui/core/Modal";
+import { Box, DialogContent, DialogTitle, IconButton } from "@material-ui/core";
+import "./styles.scss";
+import { GridCloseIcon } from "@material-ui/data-grid";
+
+function getModalStyle() {
+  const top = 50;
+  const left = 50;
+  return {
+    top: `${top}%`,
+    left: `${left}%`,
+    transform: `translate(-${top}%, -${left}%)`,
+    display: "inline-block",
+    width: "auto",
+  };
+}
```

Comment:
If top and left are static you could add this in a scss file.

Reply:
I applied it